static/app/components/forms/controls/rangeSlider/index.tsx

```diff
@@ -1,12 +1,14 @@
 import {forwardRef as reactFowardRef, useEffect, useState} from 'react';
-import styled from '@emotion/styled';
 
 import Input from 'sentry/components/forms/controls/input';
 import {t} from 'sentry/locale';
-import space from 'sentry/styles/space';
 import {defined} from 'sentry/utils';
 
-type Props = {
+import Slider from './slider';
+import SliderAndInputWrapper from './sliderAndInputWrapper';
+import SliderLabel from './sliderLabel';
+
+type SliderProps = {
```

Comment:
Should this be `RangeSliderProps` (since we have a different `Slider` component)?